src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler.Transport/Microsoft.CodeAnalysis.Razor.Compiler.Transport.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.Build.NoTargets/2.0.1">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>Transport package for Microsoft.CodeAnalysis.Razor.Compiler assemblies. For internal use only.</Description>
+    <IsShipping>false</IsShipping>
+    <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <IsPackable Condition="'$(OS)' == 'Windows_NT'">true</IsPackable>
```

Comment:
Why do we need this? Would expect it to be packagable on any OS as there is nothing particularly OS specific here.

Reply:
I wasn't sure how to create a transport package, so I copied this from the source generator csproj.

Reply:
Just saw this PR which does the same thing, so it's probably correct? https://github.com/dotnet/razor/pull/8397/files

Reply:
@allisonchou could you comment on this? Are we going to have a similar problem with the new Microsoft.CodeAnalysis.Razor.Compiler package?

It somewhat feels like this should only be conditional in CI. I would like to be able to pack things locally on my Linux dev box.

Reply:
Hm, there's even older PR which also does the same thing; but comments under it suggest we don't want that: https://github.com/dotnet/razor/pull/8275#issuecomment-1444589153 - I'm confused

Reply:
Sorry for the late reply here. We limit to only Windows so we don't publish duplicate packages to darc. However, we could definitely change this to be conditional on CI only - does anyone happen to know the correct condition variable for that?

Reply:
It looks like [Roslyn](https://github.com/dotnet/roslyn/blob/main/eng/build.ps1#LL273C8-L273C34) uses `ContinuousIntegrationBuild`, but we haven't set it up in Razor

Reply:
There's a new PR that seems to be solving this: https://github.com/dotnet/razor/pull/8426

Reply:
Merged main with that PR here, and removed the condition from this new transport project, as well.